src/commands/open.ts

```
@@ -19,23 +20,80 @@ export const COMMAND = "open";
+		args.push("--output", config.open.buildOutputPath);
+	}
+
+	if (config.open.projectPath !== undefined) {
+		args.push("--project", config.open.projectPath);
+	}
+
+	return args;
+}
+
+async function runOpenBuild(config: ResolvedConfig): Promise<void> {
+	if (config.projectType === "rbxts") {
```

ESLint

[error] Unsafe member access .projectType on a type that cannot be resolved.

(ts/no-unsafe-member-access)